.github/workflows/auto-merge.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   set-auto-merge:
-    runs-on: ubuntu-latest
```

Comment:
Please use latest everywhere. This is for example the label used by our self hosted runners.